cmd/geth/main.go

```diff
@@ -52,6 +52,11 @@ var (
 	app = utils.NewApp(gitCommit, "the go-ethereum command line interface")
 	// flags that configure the node
 	nodeFlags = []cli.Flag{
+		utils.MaxNoFileFlag,
```

Comment:
Can we rename this to `MaxNumFileFlag`? `No` is used as an abbreviation for Number here, but it is also used as a the word `No` in other fields, which causes confusion

Reply:
got it.